src/lightning_app/runners/cloud.py

```diff
@@ -107,10 +117,41 @@ def dispatch(
                     preemptible=work.cloud_compute.preemptible,
                     shm_size=work.cloud_compute.shm_size,
                 )
+
+                drive_specs: List[V1LightningworkDrives] = []
+                for drive in work.drives:
+                    if drive.protocol == "lit://":
```

Comment:
I am curious, why passing the lit information to the specs ? Does the platform does something special there.

Reply:
@tchaton not presently, but we could do some interesting things later down the line ;)

Mounting or cloning lit drives, monitoring usage etc.